pkg/apis/provisioning/v1alpha5/constraints.go

```diff
@@ -37,6 +37,9 @@ type Constraints struct {
 	// KubeletConfiguration are options passed to the kubelet when provisioning nodes
 	//+optional
 	KubeletConfiguration KubeletConfiguration `json:"kubeletConfiguration,omitempty"`
```

Comment:
Can we add a comment that points to https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration and explains that this struct is a subset of this configuration?